.github/workflows/tests-prql-query.yml

```diff
@@ -0,0 +1,23 @@
+name: tests
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  linux-install:
+    runs-on: ubuntu-latest
+    steps:
+      - run: /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+      - run: eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
+      - run: brew tap prql/homebrew-prql
+      - run: brew install prql-query
```

Comment:
```suggestion
      - run: brew install prql/prql/prql-query
```

Reply:
It's a shorter syntax for installing formulas from taps; released in the past year or two I think